sql/moz-fx-data-shared-prod/firefox_ios_derived/app_store_funnel_web_referrer_v1/query.sql

```diff
@@ -0,0 +1,366 @@
+    NULLIF(campaign, "MISSING") AS campaign,
+    COALESCE(impressions, 0) AS impressions,
+    COALESCE(impressions_unique_device, 0) AS impressions_unique_device,
+    COALESCE(page_views, 0) AS page_views,
+    COALESCE(page_views_unique_device, 0) AS page_views_unique_device,
+    COALESCE(first_time_downloads, 0) AS first_time_downloads,
+    COALESCE(redownloads, 0) AS redownloads,
+    COALESCE(total_downloads, 0) AS total_downloads,
+    COALESCE(active_devices, 0) AS active_devices,
+    COALESCE(active_devices_last_30_days, 0) AS active_devices_last_30_days,
+    COALESCE(deletions, 0) AS deletions,
+    COALESCE(installations, 0) AS installations,
+    COALESCE(sessions, 0) AS sessions,
+  FROM
+    views_territory
+  FULL OUTER JOIN
+    downloads_territory
+    USING (`date`, app_id, territory, web_referrer, campaign)
+  FULL OUTER JOIN
+    usage_territory
+    USING (`date`, app_id, territory, web_referrer, campaign)
+)
+SELECT
+  `date`,
+  app_id,
+  territory,
+  source_type,
+  app_referrer,
+  web_referrer,
+  NULLIF(campaign, "MISSING") AS campaign,
+  COALESCE(
+    web_referrer_stats.impressions,
+    app_referrer_stats.impressions,
+    source_type_stats.impressions,
+    0
+  ) AS impressions,
+  COALESCE(
+    web_referrer_stats.impressions_unique_device,
+    app_referrer_stats.impressions_unique_device,
+    source_type_stats.impressions_unique_device,
+    0
+  ) AS impressions_unique_device,
+  COALESCE(
+    web_referrer_stats.page_views,
+    app_referrer_stats.page_views,
+    source_type_stats.page_views,
+    0
+  ) AS page_views,
+  COALESCE(
+    web_referrer_stats.page_views_unique_device,
+    app_referrer_stats.page_views_unique_device,
+    source_type_stats.page_views_unique_device,
+    0
+  ) AS page_views_unique_device,
+  COALESCE(
+    web_referrer_stats.first_time_downloads,
+    app_referrer_stats.first_time_downloads,
+    source_type_stats.first_time_downloads,
+    0
+  ) AS first_time_downloads,
+  COALESCE(
+    web_referrer_stats.redownloads,
+    app_referrer_stats.redownloads,
+    source_type_stats.redownloads,
+    0
+  ) AS redownloads,
+  COALESCE(
+    web_referrer_stats.total_downloads,
+    app_referrer_stats.total_downloads,
+    source_type_stats.total_downloads,
+    0
+  ) AS total_downloads,
+  COALESCE(
+    web_referrer_stats.active_devices,
+    app_referrer_stats.active_devices,
+    source_type_stats.active_devices,
+    0
+  ) AS opt_in_active_devices,
+  COALESCE(
+    web_referrer_stats.active_devices_last_30_days,
+    app_referrer_stats.active_devices_last_30_days,
+    source_type_stats.active_devices_last_30_days,
+    0
+  ) AS opt_in_active_devices_last_30_days,
+  COALESCE(
+    web_referrer_stats.deletions,
+    app_referrer_stats.deletions,
+    source_type_stats.deletions,
+    0
+  ) AS deletions,
+  COALESCE(
+    web_referrer_stats.installations,
+    app_referrer_stats.installations,
+    source_type_stats.installations,
+    0
+  ) AS installations,
```

Comment:
```suggestion
  ) AS opt_in_deletions,
  COALESCE(
    web_referrer_stats.installations,
    app_referrer_stats.installations,
    source_type_stats.installations,
    0
  ) AS opt_in_installations,
```

Per https://developer.apple.com/help/app-store-connect/reference/app-metrics